### PR TITLE
Add user mapping chapter

### PR DESCRIPTION
**Pull request by dmpop**
### Description

I removed the user mapping content from Troubleshooting and replaced it with a dedicated topic (aka chapter).

### CODE REVIEW

File: concepts/rootless-podman-user-mapping.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  <section xml:id="rootless-containers-user-mapping">
+    <title>User mapping and rootless containers</title>
+    <para>
+      By definition, rootless containers are run by a regular user. At the same
+      time, certain applications deployed using containers expect to be run as
+      root. This leads to a problem: how do you run a container as root, when
+      you are not root on the host system? To solve the issue, &podman; relies
+      on user namespaces to map user IDs in the container to different user IDs
+      on the host. By default, &podman; maps the root user inside the container
+      with the user ID (UID) 0 to the UID of current user on the host system.
```
Comment (dariavladykina) [inline concepts/rootless-podman-user-mapping.xml:27]:
```suggestion
      with the user ID (UID) 0 to the UID of the current user on the host system.
```

File: concepts/rootless-podman-user-mapping.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    </para>
+    <para>
+      To illustrate how this is done in practice, create a temporary
+      &bcia;-Base container (the <literal>sleep</literal> value determines for
+      how long container runs):
```
Comment (dariavladykina) [inline concepts/rootless-podman-user-mapping.xml:32]:
```suggestion
      how long the container runs):
```

File: concepts/rootless-podman-user-mapping.xml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    <para>
+      Then run the <command>podman top</command> command as follows:
+    </para>
+<screen>&prompt.user;podman top -l user huser
+USER  HUSER
+root  1000</screen>
+    <para>
+      The output indicates that the root user in the container is mapped to the
+      user with UID 1000 on the host, so a root process inside the container is
+      treated by kernel as UID 1000 outside the container. This means that even
```
Comment (dariavladykina) [inline concepts/rootless-podman-user-mapping.xml:44]:
```suggestion
      treated by the kernel as UID 1000 outside the container. This means that even
```

File: concepts/rootless-podman-user-mapping.xml [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      though the process is running as root inside the container, this process
+      does not have root privileges outside of the container. In addition to
+      that, if a file owned by a UID that is not mapped into the user namespace
+      is treated as owned by "nobody" (UID 65534), and the container process is
```
Comment (dariavladykina) [inline concepts/rootless-podman-user-mapping.xml:48]:
```suggestion
      is treated as owned by <quote>nobody<quote> (UID 65534), and the container process is
```

Comment (dariavladykina) [reply concepts/rootless-podman-user-mapping.xml:48]: I have a bit of a problem parsing this sentence. Probably **and** after the comma is extraneous? 
_[...] if a file [...] is treated as owned by “nobody” (UID 65534), and the container process may not access the object, unless the object is world readable and writable._

**Review by dariavladykina — COMMENTED** [review]
Hi Dima, please see some suggestions here. Thanks!

**Comment by fsundermeyer** [comment]
Replaced by #284

**Pull request closed without merging**